stress.yaml

```diff
@@ -27,19 +27,32 @@
     - set_fact: index={{ec2_tag_index}}
       when: index is not defined
 
+    - set_fact: taskset="taskset -c {{load_cpus}}"
+      when: load_cpus is defined
+
+    - set_fact: taskset=""
+      when: load_cpus is not defined
```

Comment:
Maybe. I had runs with and without it, so from a dev PoV, it is definitely nice to be able to take this in and out.

But as for the finished thing, if we establish that this is always the best way, we can certainly make it a property of the node.